Packs/ProofpointThreatResponse/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Proofpoint Threat Response (Beta)
+Maintenance and stability enhancements.
```

Comment:
I think on this occasion we need more detailed release notes

Reply:
Done

Reply:
Not seeing the  changes

Reply:
Somehow it was not appear here. Did push again. (you can see it below)